examples/resnet/resnet_vitis_ai_ptq_cpu.json

```diff
@@ -43,7 +43,14 @@
                             "goal": { "type": "max-degradation", "value": 0.1 }
                         }
                     ],
-                    "user_config": { "user_script": "user_script.py", "evaluate_func": "eval_accuracy" }
+                    "user_config": {
```

Comment:
since evaluate func is used, I think the `data_config` is not needed here?

Reply:
Removed.